Release/include/pplx/pplx.h

```diff
@@ -133,7 +133,7 @@ class _TaskCollectionImpl
 public:
     typedef _TaskProcHandle _TaskProcHandle_t;
 
-    _TaskCollectionImpl(scheduler_ptr _PScheduler) : _M_pScheduler(_PScheduler) {}
+    _TaskCollectionImpl(scheduler_ptr _PScheduler) : _M_pScheduler(_PScheduler), _M_scheduled_task(nullptr) {}
```

Comment:
would it make sense to just set this down on line 201 instead?  Just thinking of minimizing the changes for future upgrades.